### PR TITLE
[LA-2024] CFP is open now

### PR DESCRIPTION
**Pull request by omargohan**
**Summary of changes:**

- Added Propose
- Added Registration
- Updated Welcome 
- Updated LA 2023 program and linked to LA 2024 Welcome page
- Small updates to be consistent in the rest of the pages

**Tested:**

- Locally using Hugo

### CODE REVIEW

File: data/events/2024-los-angeles.yml [hunk 1]
```diff
@@ -44,7 +44,9 @@ nav_elements: # List of pages you want to show up in the navigation of your page
   # - name: CFP
   #   url: https://www.socallinuxexpo.org/scale/21x/cfp
```
Comment (phrawzty) [inline data/events/2024-los-angeles.yml:45]: Up to you, but we recommend it for SEO etc.

```suggestion
  - name: CFP
    url: https://www.socallinuxexpo.org/scale/21x/cfp
```

**Review by phrawzty — COMMENTED** [review]
Just nits and I'll merge, but you may wish to address these items in a future PR. 🙂

File: data/events/2024-los-angeles.yml (hunk 1 above)
Comment (omargohan) [reply data/events/2024-los-angeles.yml:45]: We decided to have a page explaining how to submit a proposal before sending people to the SCALE website. Thank you for the recommendation, though. :)

File: content/events/2024-los-angeles/about.md [hunk 2]
```diff
@@ -3,9 +3,9 @@ Title = "About"
 Type = "event"
 Description = "About DevOpsDays Los Angeles 2024"
 +++
-
 <br>
-
+<hr>
+<h3>About Us</h3>
 <p>
 We are proud to announce the 13th Annual DevOpsDaysLA will be held Friday, March 15th, 2024. DevOpsDaysLA is an anual event held in Southern California. The conference is volunteer-organized and dedicated to the DevOps community. DevOpsDaysLA is presented in cooperation with the Southern California Linux Expo (SCALE).
```
[the remark this answers is not in the document]
Comment (omargohan) [reply content/events/2024-los-angeles/about.md:10]: Good catch. I have to update a few more items, but I'll make sure to fix this typo. Thank you!

**Pull request merged**